src/lib/endpoints.js

```diff
@@ -7,7 +7,7 @@ export const useFeedstocks = () => {
   const { data, error } = useSWR(
     `https://${orchestratorEndpoint}/feedstocks/`,
     jsonFetcher,
-    { refreshInterval: 300000 }
```

Comment:
It might be worth throwing this value into a constant with a comment to document (i.e. explain how this allows navigation between recipe runs within 2 minutes without having to re-fetch feedstock data)

Reply:
Addressed in https://github.com/pangeo-forge/pangeo-forge.org/pull/149